src/PortToTripleSlash/src/libraries/RoslynTripleSlash/TripleSlashSyntaxRewriter.cs

```diff
@@ -501,38 +507,65 @@ private static SyntaxTriviaList GetLeadingTrivia(SyntaxNode node)
             return new();
         }
 
-        private static SyntaxTriviaList GetSummary(DocsAPI api, SyntaxTriviaList leadingWhitespace)
+        // Collects all tags with of the same name from a SyntaxTriviaList.
+        private static SyntaxTriviaList FindTag(string tag, SyntaxTriviaList leadingWhitespace, SyntaxTriviaList from)
+        {
+            List<XmlNodeSyntax> l = new();
+            foreach(var trivia in from)
+            {
+                SyntaxNode? structure = trivia.GetStructure();
+                if (structure is DocumentationCommentTriviaSyntax st) {
+                    foreach(XmlNodeSyntax elem in st.Content)
+                    {
+                        if (elem is XmlEmptyElementSyntax ees && ees.Name.ToString() == tag) {
+                            l.Add(elem);
+                        } else if (elem is XmlElementSyntax es && es.StartTag.Name.ToString() == tag) {
+                            l.Add(elem);
```

Comment:
Same request about naming. Plus the first item is a node, not an element:
```suggestion
                    foreach(XmlNodeSyntax node in structure.Content)
                    {
                        if (node is XmlEmptyElementSyntax emptyElement && emptyElement.Name.ToString() == tag) {
                            list.Add(node);
                        } else if (node is XmlElementSyntax element && element.StartTag.Name.ToString() == tag) {
                            list.Add(node);
```